# joystick parameters for playstation 4 controller
dev: "/dev/input/js0" # change this to your joystick device path

# joystick parameters for src/joy_teleop.py
deadman_switch: 5 # button index
lin_vel_button: 4
ang_vel_button: 0 
